samples/tutorial/src/Binding/GetValueSample.cs

```diff
@@ -0,0 +1,24 @@
+using System.CommandLine;
+
+namespace Binding
+{
+    public static class GetValueSample
+    {
+        public static int GetValueFromOptionArgument()
+        {
+            #region GetValueFromOptionArgument
+
+            var option = new Option("--an-int")
+            {
+                Argument = new Argument<int>()
+            };
+            var parseResult = option.Parse("--an-int 123");
+            var value = parseResult.FindResultFor(option).GetValueOrDefault();
+            value.PrintToConsole();
```

Comment:
You don't consistently use PrintToConsole, so it feels a bit distracting. 

```
Console.WriteLine(FormatOutput(value));
```

would leave focus on the Ain work.